### DISCUSSION TITLE
Specify an expression using the GO API

### DISCUSSION THREAD

**Discussion by uris77** [cue-lang/cue#1214] [Q&A] [answered]
I have the following code that exports a schema to JSON:

```
	cwd, err := os.Getwd()
	if err != nil {
		return "", err
	}
	cueFilesDir := filepath.Join(cwd, "dashboards")
	dirCfg := &load.Config{
		Dir: cueFilesDir,
		//Package: "grafana",
		Tags: []string{
			fmt.Sprintf("title=%s", name),
			fmt.Sprintf("projectIds=%s", strings.Join(projectIDs, "|")),
		},
	}

	bps := load.Instances([]string{
		".",
		fmt.Sprintf("--expression=%s", strings.ToLower(name))},
		dirCfg)
	v := ct.BuildInstance(bps[0])
	byts, err := v.MarshalJSON()
```
	
I basically want to replicate this command:

`cue export --expression=pubsub --inject title="PubSub" --inject projectIds="tl-bm5pnfhi6ocyc1ogl8eo|tl-d1wvcsuk2aoug7fs0nll|tl-re9dlnolf5oa7iyk2fh8" -p grafana > dashboard.json`

where `title` == `name`. 

The cli outputs a JSON file the way I want it, but the GO code pre-fixes the name of the expression to the output. How 
can I get the same output as the cli?

**Comment by verdverm** [comment, marked as answer]
Use `LookupPath` and `ParsePath` to extract sub values from another value

- https://pkg.go.dev/cuelang.org/go@v0.4.0/cue#Value.LookupPath
- https://pkg.go.dev/cuelang.org/go@v0.4.0/cue#ParsePath

`sub := v.LookupPath(cue.ParsePath("path.to.sub"))`

**Reply by uris77** [reply]
Thank you.
